airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/manifest.yaml

```diff
@@ -8,10 +8,6 @@ check:
     - Tags
 
 definitions:
-  schema_loader:
-    type: JsonFileSchemaLoader
-    file_path: "./source_zendesk_support/schemas/{{ parameters.get('schema_name') or parameters['name'] }}.json"
```

Comment:
`schema_name` didn't seem to be defined here and `parameters['name']` is the default behavior